lib/ansible/galaxy/dependency_resolution/providers.py

```diff
@@ -13,13 +13,6 @@
 except ImportError:
     TYPE_CHECKING = False
 
-if TYPE_CHECKING:
```

Comment:
Why did you move this block? It's best not to touch parts that are not really related. Maybe you could separate this into two if-blocks?

Reply:
Because pylint complain about that if keeping original ordering.